manifests/plugin/python.pp

```diff
@@ -79,7 +79,7 @@
 
   concat::fragment { 'collectd_plugin_python_conf_header':
     order   => '00',
-    content => template('collectd/plugin/python/header.conf.erb'),
+    content => epp('collectd/plugin/python/header.conf'),
```

Comment:
Please make the extension .epp

Reply:
It's weird how it even works without. .epp added to that and other two places now.